parsec/core/types/manifest.py

```diff
@@ -213,19 +112,17 @@ class LocalManifestType(Enum):
     LOCAL_USER_MANIFEST = "local_user_manifest"
 
 
-LocalManifestTypeVar = TypeVar("LocalManifestTypeVar", bound="BaseLocalManifest")
 LocalFileManifestTypeVar = TypeVar("LocalFileManifestTypeVar", bound="LocalFileManifest")
 LocalFolderManifestTypeVar = TypeVar("LocalFolderManifestTypeVar", bound="LocalFolderManifest")
 LocalWorkspaceManifestTypeVar = TypeVar(
     "LocalWorkspaceManifestTypeVar", bound="LocalWorkspaceManifest"
 )
 LocalUserManifestTypeVar = TypeVar("LocalUserManifestTypeVar", bound="LocalUserManifest")
-LocalManifestsTypeVar = Union[
-    LocalManifestTypeVar,
-    LocalFileManifestTypeVar,
-    LocalFolderManifestTypeVar,
-    LocalWorkspaceManifestTypeVar,
-    LocalUserManifestTypeVar,
+LocalManifestTypeVar = Union[
+    LocalFileManifest,
+    LocalFolderManifest,
+    LocalWorkspaceManifest,
+    LocalUserManifest,
 ]
 
 
```

Comment:
Why do we still need `BaseLocalManifest` ?

If we keep using it, we still need all the python manifests classes, and end up mixing rust and python manifest classes (for instance creating a `parsec.api.dataFileManifest` creates a rust FileManifest, while deserializing with `parsec.api.data.BaseManifest.decrypt_verify_and_load` returns a python FileManifest...)

This can lead to subtle difference in behavior between the two classes that we'd better avoid

Reply:
`BaseLocalManifest` and `BaseManifest` will be removed in a latter PR